src/ert/_c_wrappers/enkf/lark_parser.py

```diff
@@ -279,6 +279,16 @@ def convert(
                             f"Could not find executable {val.value!r}",
                             config_file=val.filename,
                         )
+                if not os.access(path, os.X_OK):
```

Comment:
Just wondering if this belongs to the issue? Also this `_convert` only returns fill path, so just not sure it this check was necessary :)

Reply:
Since the new and old parser are run against the same tests, unless we specifically say it is only for the old/new parser, it's best to update both at the same time.

Reply:
@xjules `_convert`?

the test i added to test the old parser basically made the new parser fail, and discovered something that should have been covered in the new parser. that's why it's included here -without this change, the test doesn't pass.

Reply:
Sure, makes sense :+1:

Reply:
ah now I see - you expect `convert` to do a very simple operation, while i added validation. @mortalisk could you suggest a better place for validation? or is this sensible, and the method should rather be renamed?